cds/modules/webhooks/receivers.py

```diff
@@ -190,17 +193,22 @@ def _init_object_version(self, event):
             ObjectVersionTag.create(object_version, 'uri_origin',
                                     event.payload['uri'])
             ObjectVersionTag.create(object_version, '_event_id', event_id)
+            # add tags for file type
+            ObjectVersionTag.create(object_version, 'media_type', 'video')
+            ObjectVersionTag.create(object_version, 'context_type', 'master')
```

Comment:
Not everything we download has necessarily to be a video, we can also enter URLs for subtitles.